app.chunker.oss.chunker.yml

```
@@ -0,0 +1,45 @@
+id: app.chunker.oss.chunker
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+base: "org.electronjs.Electron2.BaseApp"
+base-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.openjdk17"]
+command: app.chunker.oss.chunker
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: ["/usr/lib/sdk/openjdk17/install.sh"]
+  - name: chunker
+    buildsystem: simple
+    build-commands:
+      [
+        "mkdir -p /app/chunker",
+        "cp -r * /app/chunker",
+        "chmod +x /app/chunker/chunker-electron",
+        "chmod +x /app/chunker/resources/chunker-cli/bin/chunker-cli",
+        "install -Dm755 chunker.sh /app/bin/app.chunker.oss.chunker",
+        "install -Dm644 app.chunker.oss.chunker.metainfo.xml /app/share/metainfo/app.chunker.oss.chunker.metainfo.xml",
+        "install -Dm644 app.chunker.oss.chunker.desktop /app/share/applications/app.chunker.oss.chunker.desktop",
+        "install -Dm644 chunker-logo.png /app/share/icons/hicolor/512x512/apps/app.chunker.oss.chunker.png",
+      ]
```

```suggestion
    build-commands:
      - "mkdir -p /app/chunker",
      - "cp -r * /app/chunker",
      - "chmod +x /app/chunker/chunker-electron",
      - "chmod +x /app/chunker/resources/chunker-cli/bin/chunker-cli",
      - "install -Dm755 chunker.sh /app/bin/app.chunker.oss.chunker",
      - "install -Dm644 app.chunker.oss.chunker.metainfo.xml /app/share/metainfo/app.chunker.oss.chunker.metainfo.xml",
      - "install -Dm644 app.chunker.oss.chunker.desktop /app/share/applications/app.chunker.oss.chunker.desktop",
      - "install -Dm644 chunker-logo.png /app/share/icons/hicolor/512x512/apps/app.chunker.oss.chunker.png",
```

let's use proper yaml syntax.